app/containers/Forms/VerificationStatement.tsx

```diff
@@ -87,38 +121,63 @@ export const VerificationStatementComponent: React.FunctionComponent<Props> = ({
               >
                 Upload New Attachment
               </FilePicker>
+              <div>
+                Only PDF formats are accepted and file size must be smaller than
+                50MB
+              </div>
             </div>
           )}
         </Col>
-        <Col xs={12} style={{ margin: "20px 0 20px" }}>
-          {application.attachmentsByApplicationId.edges.map(({ node }) => {
-            return (
-              <>
-                <div className="attachment-link" key={node.id}>
-                  <Link href={getAttachmentDownloadRoute(node.id)} passHref>
-                    {node.fileName}
-                  </Link>{" "}
-                  <FontAwesomeIcon
-                    icon={faTrash}
-                    onClick={() => {
-                      const { environment } = relay;
-                      return deleteAttachmentMutation(environment, {
-                        connections: [
-                          application.attachmentsByApplicationId.__id,
-                        ],
-                        input: {
-                          id: node.id,
-                        },
-                      });
-                    }}
-                  />
-                </div>
-                <div className="uploaded-on">
-                  Uploaded on {dateTimeFormat(node.createdAt, "days_string")}
-                </div>
-              </>
-            );
-          })}
+        <Col
+          xs={12}
+          style={{
+            margin: "20px 0 20px",
+            display: "flex",
+            justifyContent: "space-around",
+            alignItems: "center",
+          }}
+        >
+          {application.attachmentsByApplicationId.edges.length > 0 && (
+            <table className="bc-table">
+              <tr>
+                <th>File</th>
+                <th>Created Date</th>
+                <th>Version Number</th>
```

Comment:
I wonder if "Version Number" could be confusing here. It could be perceived as the file's version number, but what we're showing here is the Application's version number. Should we be explicit here and make the header "Application Version Number"?